.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+**Description**
+A description of the PR, should include a decent explanation as to why this change was needed and a decent explanation as to what this change does
+
+**Issue**
+A link to a github issue or SEAB- ticket (using that as a prefix)
+
+Please make sure that you've checked the following before submitting your pull request. Thanks!
+
+- [ ] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
+- [ ] If the PR has any new pages, only **AFTER** the PR is approved by all reviewers, generate a new discourse topic, by running the action at https://github.com/dockstore/dockstore-documentation/actions/workflows/add-discourse-topic.yml. This is irreversible, which is why you want to ensure the new pages are approved before generating the topic.
```

Comment:
Is it irreversible?
I'd kind of assumed that all community leads could delete topics on discourse anyway, in which case creating it before would expose any other kind of build issues with new pages

Reply:
That said, clicking into the build I can see the link check and build worked
If we stick with this approach, suggesting new ticket for @ll5zh to split out just the discourse topic check into a new build so we can see at a glance the rest of the build succeeded

Reply:
> Is it irreversible? I'd kind of assumed that all community leads could delete topics on discourse anyway, in which case creating it before would expose any other kind of build issues with new pages

Irreversible was too strong a word. I'll update the text.

Reply:
> That said, clicking into the build I can see the link check and build worked If we stick with this approach, suggesting new ticket for @ll5zh to split out just the discourse topic check into a new build so we can see at a glance the rest of the build succeeded

SEAB-6142